pkg/cli/webhook.go

```diff
@@ -87,8 +87,13 @@ func (c *WebhookServerCommand) RunUnstarted(ctx context.Context, args []string)
 	}
 	logger.Debugw("loaded configuration", "config", c.cfg)
 
-	pubsubClientOpts := append([]option.ClientOption{option.WithUserAgent("abcxyz/github-metrics-aggregator")}, c.testPubSubClientOptions...)
-	webhookServer, err := webhook.NewServer(ctx, c.cfg, pubsubClientOpts...)
+	opts := append([]option.ClientOption{option.WithUserAgent("abcxyz/github-metrics-aggregator")}, c.testPubSubClientOptions...)
```

Comment:
As a follow-up, it would be good to set the user-agent to include a version, something like `abcxyz:github-metrics-aggregator/1.2.0`. I realize this is existing code.